trappy/base.py

```diff
@@ -206,6 +208,18 @@ def generate_data_dict(self, data_str):
         prev_key = None
         for field in data_str.split():
             if "=" not in field:
+                if not prev_key:
+                    if 'FAILED TO PARSE' in data_str:
+                        warnings.warn(
+                            'trace-cmd failed to parse the "{}" event. You may '
```

Comment:
Ideally we'd want to have each warning only once (one per failed to parse event). Can we just roll with the "warnings only printed once" thing or should we do something a bit better ?

Reply:
I think it only prints it once for each event already

Reply:
LGTM then